### DISCUSSION TITLE
Welcome to send-pdf-for-contact-form-7 Discussions!

### DISCUSSION THREAD

**Discussion by restezconnectes** [Florent73/send-pdf-for-contact-form-7#59] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shasha-24** [comment]
Bonjour M. Maillefaud,

Je vous contacte concernant votre plugin Send PDF for Contact Form 7 que j'utilise actuellement.

Lorsque je génère un PDF via mon formulaire Contact Form 7, tous les champs apparaissent dans le document, même ceux qui n'ont pas été remplis par l'utilisateur. J'aimerais masquer ces champs vides pour avoir un PDF plus propre.

Ce que j'ai testé :

- J'ai installé la version GitHub du plugin comme mentionné dans votre tutoriel
- J'ai cherché dans tous les paramètres mais je ne trouve pas l'option pour masquer les champs vides

Pourriez-vous m'indiquer où se trouve cette fonctionnalité ou me donner la marche à suivre ?

Je vous remercie par avance pour votre aide.
